front/src/pages/Dashboard/Dashboard.styles.tsx

```diff
@@ -35,3 +43,8 @@ export const CardMonitor = styled.div`
   box-shadow: -1px 12px 23px -10px rgb(0 0 0 / 15%);
   border: 1px solid #eaeaea;
 `;
+
+export const Container = styled.div`
+  max-width: 1200px;
```

Comment:
1200px can be a const, this to improve the manutentability of code.

Reply:
added